Core/DISET/private/Transports/BaseTransport.py

```diff
@@ -25,6 +25,7 @@
 
 import time
 import select
+import six
```

Comment:
changing this file makes me kind of nervous :-) Anyway, it has to be done. 
I just hope we won't get tricked with non UTF character since then `len` comparison would not make sense anymore..

Reply:
> changing this file makes me kind of nervous :-)

Wait until you see my next PR...

>  I just hope we won't get tricked with non UTF character since then len comparison would not make sense anymore..

This won't be a problem as this treats everything as a sequence of bytes instead of a string so the length will always be correct.

Reply:
> Wait until you see my next PR...
Now you scare me :-) 

>This won't be a problem as this treats everything as a sequence of bytes instead of a string so the length will always be correct.
Yeah that's what I understood, but since I still have a hard time getting my head around that.. :-P